### ISSUE TITLE
version 8.x.x

### ISSUE THREAD

**Issue by dfroger** [conda-forge/nodejs-feedstock#14]
Would it be a possible to upgrade nodejs to version 8.x.x, instead of 6.x.x? (I'll try to make a PR).

**Comment by dfroger** [comment]
https://nodejs.org/en/ v6.11.2 is LTS  and "recommanded for most users", v8.4.0 is the current with "lastest features".

May be interessing to provide both.

**Comment by minrk** [comment]
Makes sense to ship 8.x. We can make a 6.x branch to track LTS.

**Comment by alipunjani** [comment]
Any plans to add node v8 now that it's LTS?

**Comment by minrk** [comment]
Yes, that's the plan. Nobody has had time to figure out what's wrong with the build on Windows, which is blocking the update.

**Comment by minrk** [comment]
#18 fixed building 8.8.1 on Windows, with #19 to update to 8.9.1. A 6.x branch has been created to get updates for 6.x releases.
